samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -5,13 +5,12 @@
 
 #include <zephyr/kernel.h>
 #include <stdio.h>
-#include <modem/nrf_modem_lib.h>
-#include <modem/lte_lc.h>
 #include <zephyr/net/socket.h>
 #include <net/nrf_cloud.h>
 #include <net/nrf_cloud_codec.h>
 #include <date_time.h>
 #include <zephyr/logging/log.h>
+#include <modem/nrf_modem_lib.h>
```

Comment:
Keeping `#include <modem/nrf_modem_lib.h>` for now until https://github.com/nrfconnect/sdk-nrf/pull/11519 merges